hal/src/stm32f2xx/usart_hal.c

```diff
@@ -262,8 +262,8 @@ void usartEndImpl(hal_usart_interface_t serial, bool cache) {
     USART_Cmd(usartMap[serial]->peripheral, DISABLE);
 
     // Switch pins to INPUT
-    HAL_Pin_Mode(usartMap[serial]->rx_pin, INPUT);
-    HAL_Pin_Mode(usartMap[serial]->tx_pin, INPUT);
+    HAL_Pin_Mode(usartMap[serial]->usart_rx_pin, INPUT);
+    HAL_Pin_Mode(usartMap[serial]->usart_tx_pin, INPUT_PULLUP);
```

Comment:
Only when going into a sleep state?

Reply:
Wouldn't be better to also optimize power consumption if USART is ended?

Reply:
In the case where we are just stopping the UART we supposedly don't have anything powered on the other side. I would assume that `INPUT_PULLUP` would consume more power than `INPUT`.